addons/stock/report/report_deliveryslip.xml

```diff
@@ -96,6 +99,9 @@
                                                 <td>
                                                     <span t-field="packlot.qty"/>
                                                 </td>
+                                                <td>
+                                                    <span class="pull-right" t-field="packlot.lot_id.life_date" t-field-options='{"format": "dd/MM/yy"}'/>
```

Comment:
how can you get this "life_date" without installing product_expiry ? 
you should inherit this report in product_expiry and add this field from that module.